app/models/concerns/public_body_derived_fields.rb

```diff
@@ -7,13 +7,13 @@ module PublicBodyDerivedFields
     before_save :set_first_letter
 
     # When name or short name is changed, also change the url name
-    def short_name=(short_name)
-      super
+    def short_name=(value)
+      write_attribute("short_name", value)
```

Comment:
Better commit message:

ActiveRecord attributes that are not bound to database columns are handled through `method_missing` so there's no named method for `super` to find. The handler is a wrapper for `write_attribute` so we can safely call that directly ourselves.

https://apidock.com/rails/v4.1.8/ActiveRecord/AttributeMethods/Write/attribute=